lib/Backend/Commands.hs

```diff
@@ -91,7 +91,19 @@ createCmd (CreateOptions entryPath _edit force tags fields privateFields) = do
       & E.fields .~ HashMap.fromList allFields
       & E.tags .~ tags
 
-  -- TODO: check that a directory does not exist at the given path.
+    checkForEntriesInEntryPath :: EntryPath -> Sem r ()
```

Comment:
Nitpick: I think it would be better to define this helper function in a `where` clause at the bottom.

I think it currently breaks the "flow" when reading this `do`-block from top to bottom.

Reply:
Done